whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -1690,4 +1693,145 @@ public void search_inetnum_single_limit_offset_by_two() {
         assertThat(response.getWhoisObjects().get(0).getPrimaryKey().get(0).getValue(), is("TP1-TEST"));
     }
 
+    // If no source is defined (the default) both "source: RIPE" and “source: RIPE-NONAUTH” ROUTE(6) objects are returned
```

Comment:
Also search for inetnum with out-of-region route, and check the route is returned as a related object.

Reply:
expanded test case 👍